### DISCUSSION TITLE
📢 Announcing Licensing and Ownership Transfer Update for the ESPHome Samsung HVAC Project

### DISCUSSION THREAD

**Discussion by omerfaruk-aran** [omerfaruk-aran/esphome_samsung_hvac_bus#211] [Announcements]
Hello everyone,

We are excited to announce an important update regarding the ESPHome Samsung HVAC integration project.

After extensive collaboration and contributions, **Steve Wagner** (known as *@lanwin*) has kindly transferred the ownership of the project to **Ömer Faruk Aran** (*@omerfaruk-aran*). This transition marks a new chapter for the project and ensures its continued growth and support for the community.

---

### 🔍 What Does This Mean?

1. **Project Origins & Acknowledgments**: This project was originally developed and launched by [lanwin (Steve Wagner)](https://github.com/lanwin). His vision and contributions laid the foundation for everything that follows. To honor his contributions, his name will be prominently displayed across the project's *README*, *LICENSE*, and other key areas as the **Project Founder**.

2. **Updated Licensing**: We’ve officially added a new *MIT License* to the project to provide a clear, open-source pathway for development. The license includes a credit acknowledging the project’s original development by [Steve Wagner](https://github.com/lanwin).

3. **New Development & Enhancements**: Moving forward, we plan to introduce a more streamlined integration approach via Home Assistant, along with additional improvements in usability and functionality. We are committed to expanding the project's scope while maintaining its open-source principles and community-centric nature.

4. **Gratitude to the Community**: We recognize and appreciate all the contributors and users who have supported the project. Your feedback, contributions, and passion are what drive this project forward.

---

### 🚀 What’s Next?

We encourage everyone to continue sharing feedback, opening issues, and contributing as we work towards making the integration even more robust and user-friendly. This project remains a **community effort**, and we look forward to collaborating with all of you.

For any questions or contributions, please feel free to reach out or open a new discussion.

Thank you once again, **Steve Wagner (@lanwin)**, for your incredible contributions and for entrusting us with this project! ❤️

Best regards,  
**Ömer Faruk Aran** (omerfaruk-aran)

---

### 🔗 Relevant Links
- [Project Repository](https://github.com/omerfaruk-aran/esphome_samsung_hvac_bus)
- [lanwin’s Profile](https://github.com/lanwin)

**Comment by mbo18** [comment]
Hi, 
Does that mean that you are moving away from ESPHome for this project?
I do not use HACS and I won’t use it. Will you keep the esphome component alive?

**Reply by omerfaruk-aran** [reply]
Hi @mbo18,

Thank you for raising this concern!

We have no intention of moving away from ESPHome for this project. In fact, the ESPHome component will remain active and fully supported. We understand that some users do not use HACS, and we want to ensure that the ESPHome integration remains robust and accessible for everyone.

The new HACS module is being developed to provide additional features and ease of configuration through Home Assistant, but this will be an optional enhancement. If you're comfortable with the existing ESPHome setup, you can continue using it just like before, without any need to transition to HACS.

Please feel free to share any additional concerns or suggestions you might have!

Best regards,
Ömer Faruk ARAN
